### ISSUE TITLE
Add working-directory input variable

### ISSUE THREAD

**Issue by belon** [mr-smithers-excellent/docker-build-push#21]
All commands in action should be relative to `working-directory` unless absolute path is specified for `dockerfile` or `directory`.
### Example repository structure
```
repo
  |_ ...
  |_ apps
     |_ app1
        |_ Dockerfile
        |_ ...
     |_ app2
        |_ Dockerfile
        |_ ...
```
### Example action configuration for app1 without working-directory
```
env:
  BASE_DIR: "apps/app1"

...
 - name: Docker build, tag, and push image
    uses: mr-smithers-excellent/docker-build-push@v3
    with:
      image: app1
      dockerfile: ${{ env.BASE_DIR }}/Dockerfile
      directory: ${{ env.BASE_DIR }}
```
### Example action configuration for app1 with working-directory
```
env:
  BASE_DIR: "apps/app1"

...
 - name: Docker build, tag, and push image
    uses: mr-smithers-excellent/docker-build-push@v3
    with:
      image: app1
      working-directory: ${{ env.BASE_DIR }}
```

**Comment by mr-smithers-excellent** [comment]
Hi @belon a very similar issue was discussed and implemented in #15. For your use case above, can you try this?

```yaml
    - name: Build app1 image
      uses: mr-smithers-excellent/docker-build-push@v3
      with:
        image: your-repo/app1
        dockerfile: 'apps/app1/Dockerfile'
        directory: 'apps/app1'

    - name: Build app2 image
      uses: mr-smithers-excellent/docker-build-push@v3
      with:
        image: your-repo/app2
        dockerfile: 'apps/app2/Dockerfile'
        directory: 'apps/app2'
```

**Comment by belon** [comment]
@Hi @mr-smithers-excellent yes this works well and I use it as a workaround. Adding `working-directory` would be a bit cleaner.
